src/feature/home/constant/about.js

```diff
@@ -29,20 +30,49 @@ export const ABOUT_SNOROSE = `스노로즈는 숙명인이 직접 운영하는 
 카카오톡 문의 창구: (http://pf.kakao.com/_Xmhxhn)`;
 
 export const SNOROSE_HISTORY = Object.freeze([
-  '스노로즈 1.0(블랙로즈) - 07.08.15 오픈',
-  '스노로즈 2.0(블루로즈) - 08.01.01 오픈',
-  '스노로즈 3.0(화이트로즈) - 09.11.01 오픈',
-  '스노로즈 4.0 - 15.11.12 오픈',
-  '스노로즈 5.0 - 18.09.08 오픈',
-  '스노로즈 6.0 - 24.10.12 오픈',
+  { name: '스노로즈 1.0(블랙로즈)', description: '- 07.08.15 오픈' },
+  { name: '스노로즈 2.0(블루로즈)', description: '- 08.01.01 오픈' },
+  { name: '스노로즈 3.0(화이트로즈)', description: '- 09.11.01 오픈' },
+  { name: '스노로즈 4.0', description: '- 15.11.12 오픈' },
+  { name: '스노로즈 5.0', description: '- 18.09.08 오픈' },
+  { name: '스노로즈 6.0', description: '- 24.10.12 오픈' },
```

Comment:
SNOROSE_HISTORY의 description에서 '-'를 제외하고, `AccordionListItem` 컴포넌트 내에서는 description 앞에 '-'를 직접 추가하여 표시하는 방법은 어떨까요? :)

```suggestion
  { name: '스노로즈 1.0(블랙로즈)', description: '07.08.15 오픈' },
  { name: '스노로즈 2.0(블루로즈)', description: '08.01.01 오픈' },
  { name: '스노로즈 3.0(화이트로즈)', description: '09.11.01 오픈' },
  { name: '스노로즈 4.0', description: '15.11.12 오픈' },
  { name: '스노로즈 5.0', description: '18.09.08 오픈' },
  { name: '스노로즈 6.0', description: '24.10.12 오픈' },
```

Reply:
props에 listName 추가 후 구분하는 것으로 수정 완료했습니다! 
1ad9dcbc4b7e32c6c93cf6685f80fd157649dbf8